Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -11,10 +11,14 @@ public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
 /// An `UIView` subclass that shows summary and preview information about a given channel.
 open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider, SwiftUIRepresentable {
     /// The data this view component shows.
-    public var content: (channel: _ChatChannel<ExtraData>?, currentUserId: UserId?) {
+    public var content: _ChatChannel<ExtraData>? {
         didSet { updateContentIfNeeded() }
     }
 
+    private var channel: _ChatChannel<ExtraData>? {
```

Comment:
Just a minor readability improvement since there used to be a label in the tuple and now it's gone. Please let me know if you think it's redundant

Reply:
I see what you mean but it's IMO not needed. We use the `content` everywhere and it's quite easy to get used to the pattern. I actually find seeing `channel` here more confusing than `content` :)

Reply:
Ok, let's keep it in sync with the rest of the codebase 👌